public/components/getting_started/components/getting_started_integrationCards.tsx

```diff
@@ -136,13 +136,13 @@ export const IntegrationCards = () => {
               icon={
                 integration.statics && integration.statics.logo && integration.statics.logo.path ? (
                   <img
-                    style={{ height: 100, width: 100 }}
+                    style={{ height: 60, width: 60 }}
```

Comment:
For the cards, I see the custom height and width. I see these are shared between getting started and integrations may be we can create a custom class in scss so both of them consume it directly. Can be added here: https://github.com/opensearch-project/dashboards-observability/blob/main/public/index.scss

Reply:
Added

```
.synopsisIcon {
    height: 40px;
    width: 40px;
  }
```